Source/Core/Core/DiscordPresence.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2008 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DiscordPresence.h"
+#include "discord-rpc/include/discord_rpc.h"
+#include <ctime>
+
+namespace Discord
+{
+void Init()
+{
+  DiscordEventHandlers handlers = {0};
```

Comment:
In C++ it's sufficient to just do `... = {};` to initialize the entire struct to zero.